### ISSUE TITLE
Custom workspace directory

### ISSUE THREAD

**Issue by karqo** [continuedev/continue#2381]
### Validations

- [ ] I believe this is a way to improve. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) that requests the same enhancement

### Problem

The workspace directory path  (`/<home>/.continue`, currently) does not seem customizable.
It would be desirable, e.g., when facing storage problems though.

### Solution

If not already implemented (if so, how?!), it would be very nice to have the possibility (e.g., via a corresponding environmental variable) to define a custom workspace root directory, other than `/<home>/.continue` .

**Comment by Patrick-Erichsen** [comment]
Hi @karqo , appreciate the feedback here. This would introduce a decent amount of complexity so I'm not sure if we'll tackle it in the near term, but it's good to know that you'd find value in making this configurable.

**Comment by karqo** [comment]
In the meantime, 
it has turned out that this actually seems to be already working via setting the environmental variable `CONTINUE_GLOBAL_DIR` appropriately ...

**Comment by Patrick-Erichsen** [comment]
Ah yes good point, we use that for testing but don't really expose/document it anywhere. Has that caused any other issues for you so far?

**Comment by karqo** [comment]
No, so far, I did not recognize any side effects yet, that I would relate to this matter.
